src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -77,6 +73,14 @@ private String computeDynamicFormatString() {
 		private int getTransactionColumnWidth() {
 			return Math.min(MAX_ROW_WIDTH - PRE_ALLOCATED_SPACE_PER_ROW, maxTransactionNameLength);
 		}
+		
+		private String getTransactionDisplayName(String transactionName) {
+			int transactionColumnWidth = getTransactionColumnWidth();
+			if (transactionName.length() > transactionColumnWidth) {
```

Comment:
Right after I pushed this commit, I said to myself "should've used a ternary"; fixed.